packages/react-table/src/demos/Table.md

```diff
@@ -3054,7 +3054,12 @@ class LoadingStateDemo extends React.Component {
             props: { colSpan: 8 },
             title: (
               <Bullseye>
-                <Spinner size="xl" aria-labelledby="loading-table-demo" />
+                <EmptyState>
+                  <EmptyStateIcon variant="container" component={Spinner} />
+                  <Title size="lg" headingLevel="h4">
```

Comment:
Looks like this should be an `<h2>` since the only other heading on the page is an `<h1>`